src/js/chartAuthoring.js

```diff
@@ -26,18 +26,56 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             },
             dataEntryPanel: {
                 type: "floe.chartAuthoring.dataEntryPanel",
-                createOnEvent: "onTemplatesLoaded",
-                container: "{chartAuthoring}.container",
+                createOnEvent: "onPieChartReady",
+                container: "{dataEntryPanel}.container",
                 options: {
                     resources: {
                         template: "{templateLoader}.resources.dataEntryPanel",
                         dataEntry: "{templateLoader}.resources.dataEntry"
+                    },
+                    modelRelay: {
+                        source: "{that}.model.dataEntries",
+                        target: "{floe.chartAuthoring.pieChart}.model.dataSet",
+                        singleTransform: {
+                            type: "fluid.transforms.free",
+                            args: ["{that}.model.dataEntries"],
+                            func: "floe.chartAuthoring.dataEntriesToPieChartData"
+                        },
+                        forward: "liveOnly",
+                        backward: "never"
+                    },
+                    events: {
+                        onCreate: "{chartAuthoring}.events.onPanelReady"
```

Comment:
Would be nice to name space event listeners so they can be overridden when needed. Also, since there are other listeners defined in ```floe.chartAuthoring.dataEntry``` component, this listener should be fired with ```priority: "last"```.

```
events: {
    "onCreate.escalate": {
        funcName: "{chartAuthoring}.events.onPanelReady.fire",
        priority: "last"
    }
}
```